tools/run_tests/performance/scenario_config.py

```diff
@@ -525,7 +534,8 @@ def scenarios(self):
                 resp_size=8 * 1024 * 1024,
                 secure=secure,
                 minimal_stack=not secure,
-                categories=inproc_categories + [SCALABLE],
+                categories=inproc_categories
+                + ([SCALABLE, SCALABLE_UNRESTRICTED] if secure else [SCALABLE]),
```

Comment:
Can't this just be `inproc_categories + [SCALABLE_UNRESTRICTED]`? The `secure` logic seems a bit like obfuscation, since the goal is to disable this on 8-core machines.

Reply:
Oh I see this is only an issue on insecure runs. I think a comment is warranted here, unless you go with one of my simpler suggestions.